cucumber/authenticators_jwt/features/support/jwt_jwks_helper.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# Utility methods for JWT and JWKs manipulation
+
+require 'openssl'
+require 'jwt'
+
+module JWTJWKsHelper
+
+  JWKS_ROOT_PATH = "/var/jwks".freeze
+
+  def init_jwks_file(file_name:)
+    jwks = { keys: [jwk.export] }
+    File.write("#{JWKS_ROOT_PATH}/#{file_name}", JSON.pretty_generate(jwks))
+  end
+
+  def issue_jwt_token(token_body:)
+    @jwt_token ||= JWT.encode(
+      token_body,
+      rsa_key,
+      "RS256",
```

Comment:
Shoudnl't it be configurable or at leat a default value. For future cases with different encoding algorithm

Reply:
Done